### PR TITLE
Pouta cannot connect

### PR DESCRIPTION
**Pull request by lvarin**
## Proposed changes

Briefly describe the changes you've made here. Remember to add a link to the [preview page](https://csc-guide-preview.rahtiapp.fi/origin/) of your branch.

## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [x] My pull request passes all tests.
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.rahtiapp.fi/origin/) (select your branch from the list).

https://csc-guide-preview.rahtiapp.fi/origin/pouta-cannot-connect/support/faq/why-cant-i-connect-to-my-vm-in-pouta/

### CODE REVIEW

File: docs/support/faq/why-cant-i-connect-to-my-vm-in-pouta.md [hunk 1]
```diff
@@ -1,5 +1,107 @@
...
+```sh
+$ ssh root@86.xxx.xxx.xxx               
+Please login as the user "cloud-user" rather than the user "root".
+```
+
+### Is a key pair configured in the Instance?
+
+It can happen that when the instance was created you forgot to add a key pair during the creation. This can be checked from the [instance page](https://pouta.csc.fi/dashboard/project/instances/).
+
+![Instance with no Key Pair](/cloud/img/instance-no-keypair.png)
```
Comment (trispera) [inline docs/support/faq/why-cant-i-connect-to-my-vm-in-pouta.md:72]: No image are displayed. I think the correct path would be `../../cloud/img/instance-no-keypair.png`

File: docs/support/faq/why-cant-i-connect-to-my-vm-in-pouta.md [hunk 2]
```diff
@@ -1,5 +1,107 @@
...
+
+It could be that there is a public key configured in the instance, but you lack the correct corresponding private key. It is possible to double check this thanks to the key's finger print. There are two ways to check which fingerprint is configured in the Virtual machine.
+
+1. Go to the [instance page](https://pouta.csc.fi/dashboard/project/instances/), find the row corresponding to your VM and write down the name of the key pair configured. Then go to the [Key Pairs](https://pouta.csc.fi/dashboard/project/key_pairs) page and find the key pair corresponding to the key pair you wrote earlier.
+
+    ![Key pair page](/cloud/img/key-pair-page.png)
+
+1. You can also see the finger print of the key from the Instance log. Go to [instance page](https://pouta.csc.fi/dashboard/project/instances/) and click in the name of the instance. Then click in the Log tab. You will need to find the lines that begin with `ci-info`. This is the output of the cloud init script.
+
+    ![Instance log](/cloud/img/log-instance.png)
```
Comment (trispera) [inline docs/support/faq/why-cant-i-connect-to-my-vm-in-pouta.md:86]: And here also 😅

**Review by trispera — COMMENTED** [review]
LGTM
Just small fixes

**Review by trispera — APPROVED** [review]

File: docs/support/faq/why-cant-i-connect-to-my-vm-in-pouta.md [hunk 3]
```diff
@@ -1,5 +1,107 @@
...
+There are several reasons that can cause problems when connecting to a VM. We will go through all the steps one by one.
 
-If the security group rules are correct and you should be able to connect, there may be a problem with the virtual machine image. You can connect to the console of the virtual machine using the VNC console feature of the web interface. Select "Console" from the dropdown menu that is on the right side of the virtual machine listing on the instances page. Depending on the image, you may need login credentials.
\ No newline at end of file
+## Network
+
+### Floating IP
+
+Before connecting to a cPouta VM, it is necessary to add a Floating IP. This step is not necessary in ePouta, indeed ePouta does not provide Virtual IPs, one must connect directly to the Private ip.
```
[the remark this answers is not in the document]
Comment (lvarin) [reply docs/support/faq/why-cant-i-connect-to-my-vm-in-pouta.md]: Yes, I will change this

File: docs/support/faq/why-cant-i-connect-to-my-vm-in-pouta.md (hunk 1 above)
Comment (lvarin) [reply docs/support/faq/why-cant-i-connect-to-my-vm-in-pouta.md]: fixed

File: docs/support/faq/why-cant-i-connect-to-my-vm-in-pouta.md [hunk 4]
```diff
@@ -1,5 +1,107 @@
...
+
+The only solution for this issue is to recreate the instance, this time remember to add a key pair. See the [SSH Key Pair](/cloud/tutorials/ssh-key/) and the [Creating a virtual machine](/cloud/pouta/launch-vm-from-web-gui/) articles.
+
+### Are you offering the correct private key?
+
+It could be that there is a public key configured in the instance, but you lack the correct corresponding private key. It is possible to double check this thanks to the key's finger print. There are two ways to check which fingerprint is configured in the Virtual machine.
+
+1. Go to the [instance page](https://pouta.csc.fi/dashboard/project/instances/), find the row corresponding to your VM and write down the name of the key pair configured. Then go to the [Key Pairs](https://pouta.csc.fi/dashboard/project/key_pairs) page and find the key pair corresponding to the key pair you wrote earlier.
+
+    ![Key pair page](/cloud/img/key-pair-page.png)
```
[the remark this answers is not in the document]
Comment (lvarin) [reply docs/support/faq/why-cant-i-connect-to-my-vm-in-pouta.md]: fixed

File: docs/support/faq/why-cant-i-connect-to-my-vm-in-pouta.md (hunk 2 above)
Comment (lvarin) [reply docs/support/faq/why-cant-i-connect-to-my-vm-in-pouta.md]: fixed

**Comment by lvarin** [comment]
All were fixed, thanks!

**Pull request merged**